bigchaindb/web/views/parameters.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+def valid_txid(txid):
+    if re.match('^[a-fA-F0-9]{64}$', txid):
+        return txid.lower()
+    raise ValueError("Invalid hash")
+
+
+def valid_bool(val):
+    if val == 'true':
+        return True
+    if val == 'false':
+        return False
+    raise ValueError('Boolean value must be "true" or "false" (lowercase)')
```

Comment:
Ok. 

It could be argued that the set of allowed operations should be represented by constants for re-use throughout the code as in https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/common/transaction.py#L412, and then various parts of the code can reuse it including the http api layer. This would save code, and allow us to fall back on the existing mechanism of the framework we use, meanwhile still offering re-usability and modularity. E.g.:

```python
 parser.add_argument('operation', type=str, choices=ALLOWED_OPERATIONS_EXT)